### PR TITLE
fix for https://github.com/resthub/todo-backbone-example/issues/12

### PR DESCRIPTION
**Pull request by ffacon**
see for https://github.com/resthub/todo-backbone-example/issues/12 for more details

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -115,6 +115,20 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+     <repository>
```
Comment (bclozel) [inline pom.xml:118]: OSS sonatype release repo is not necessary; it is in sync with central.
Could you just let the oss sonatype snapshot repo reference?

File: pom.xml [hunk 2]
```diff
@@ -115,6 +115,20 @@
...
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
     </repositories>
 
+	<properties>
+		<resthub.version>2.1.0</resthub.version>
```
Comment (bclozel) [inline pom.xml:131]: Could you update this line with the latest resthub version? (2.1.1)

File: pom.xml (hunk 1 above)
Comment (got5) [inline pom.xml:118]: I now understand better. Thanks for the feedback Brian.

**Comment by got5** [comment]
Since https://github.com/resthub/todo-backbone-example/pull/14, I guess that snasphot is now available at https://oss.sonatype.org/content/repositories/orgresthub-793. So I will close the pull request.
Thanks for the support.
François

**Pull request closed without merging**